src/app/components/AppLogo/AppLogo.tsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import LogoIcon from "../../assets/Icons/logoIcon";
+
+import styles from "./AppLogo.module.css";
+
+type AppLogoProps = {
+  title: string;
+};
+
+export default function AppLogo({ title }: AppLogoProps): JSX.Element {
```

Comment:
are you reusing this component throughout the app with different titles?
if not, maybe the title doesn't need to be dynamic here. then you wouldn't need props.

Reply:
Thx for pointing this out, actually I don't reuse it elsewhere. I deleted the props.